.gitignore

```diff
@@ -12,8 +12,8 @@
 node_modules/*
 bower_components/*
 
-# Development user data (settings, database, media, search index, static files)
-development/*
+# Local user data (settings, database, media, search index, static files)
+openslides_data/*
```

Comment:
I think the new directory name `openslides_data` is not very ideal because the `openslides` directory in a git checkout is very similar (and we have to open this `openslides` directory very often in command line during development). So my suggestion: rename to `data`.